container-images/scripts/llama-server.sh

```diff
@@ -9,15 +9,13 @@ if [ -n "${MODEL_CHAT_FORMAT}" ]; then
 fi
 
 if [ -n "${MODEL_PATH}" ]; then
-    eval llama-server \
-        --model "${MODEL_PATH}" \
-        --host "${HOST:=0.0.0.0}" \
-        --port "${PORT:=8001}" \
-        --gpu_layers "${GPU_LAYERS:=0}" \
-	"${CHAT_FORMAT}"
-    exit 0
+    MODEL_PATH="/mnt/models/model.file"
```

Comment:
the issue is that the condition

```
if [ -n "${MODEL_PATH}" ]; then
    MODEL_PATH="/mnt/models/model.file"
fi
```

is never reached as the image has env variable defined in its Containerfile

https://github.com/containers/ramalama/blob/main/container-images/ramalama/Containerfile#L3

so either:

1. llama-cpp-server / whisper-image should unset this default env var binding 
or
2. no need to check the case where MODEL_PATH is undefined as it's always defined

Reply:
Well  not all of the Containerfiles have that set, I am thinking of removing the MODEL_PATH setting from the Containerfiles that do have it.

Also this would allow users to specify an alternative MODEL_PATH environment with ramalama run --env MODEL_PATH=/PATH

Reply:
yes that would be the best way